Client/Clique/Common/Extensions/UIViewController+Extension.swift

```diff
@@ -10,38 +10,30 @@ import NVActivityIndicatorView
 
 extension UIViewController {
     
-    func sendOkAlert(
-        with title: String = "",
-        message: String
+    func showAlert(
+            title: String = "",
+            message: String,
+            actions: [UIAlertAction],
+            animated: Bool = true
     ) {
-        let alertController = UIAlertController(title: title,
-                                                message: message ,
-                                                preferredStyle: .alert)
-        let okAction = UIAlertAction(title: Constants.Alerts.defaultOKActionTitle,
-                                     style: .cancel)
-        alertController.addAction(okAction)
-        present(alertController, animated: true)
-    }
-    
-    func sendOKCancelAlert(
-        with title: String = "",
-        message: String,
-        actions: [UIAlertAction]
-    ) {
-        let alertController = UIAlertController(title: title,
-                                                message: message,
-                                                preferredStyle: .alert)
-        actions.forEach { alertController.addAction($0) }
-        present(alertController, animated: true)
+            let alertController = UIAlertController(
+                title: title,
+                message: message,
+                preferredStyle: .alert
+            )
+
+            actions.forEach { alertController.addAction($0) }
+            present(alertController, animated: animated)
     }
-    func sendAlert(
-        with title: String = "",
-        message: String,
-        action: UIAlertAction
+
+    func showOKAlert(
```

Comment:
I would suggest going with `showAlert` name, because if you pass the as `actionTitle: "NotOK"`, it is not OKAlert anymore 😄

```suggestion
    func showAlert(
```

Reply:
There is already function showAlert. (I copied this all from Petar 😄). So I'm not sure how to change this.

Reply:
I know you heard about it, but forgot it, so... it is called method overloading; https://www.pluralsight.com/guides/overload-methods-invoking-overload-methods-csharp

You can have two methods with the same name, but different parameters;
```swift
func method(param1: Bool)
func method(param1: Bool, param2: String)
```